assets/src/application/ui/forms/espressoForm/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const dateFormat = 'yyyy-MM-dd';
+
+export const timeFormat = 'HH:mm:ss';
```

Comment:
might want to move these to: `assets\src\application\constants\date-fns-formats.ts`

OR utilize the `dateTimeFormats` property from `useConfigData()`

Reply:
Making `EspressoForm` dependent upon EDTR config is not a good idea.

Reply:
the constants I suggested and the config are both part of the application, making them globally available to all domains.
but `EspressoForm` is also part of the application layer anyways and therefore would not be dependent on `/domain/eventEditor/` because of dependencies within its same layer.

Later on down the road when we are building out the UI for another use case that requires forms (maybe the registration form on the frontend of the site) we will still have anything in the application and infrastructure layers available to us. 

What we need to avoid doing is making logic outside of a domain use case dependent on logic within a particular domain (like the EDTR).
ie, we can NOT have the registration form on the frontend of the site dependent on the Event Editor!